index.html

```diff
@@ -0,0 +1,611 @@
+              <figure class="app-review-media">
+                <img src="/img/review-image-placeholder.png" alt="Image of Oscar Mendis" title="Oscar Mendis">
+                <figcaption>
+                  <h2>Oscar Mendis</h2>
+                  <p>Director: BoxApp Media Inc.</p>
+                </figcaption>
+              </figure>
+            </li>
+            <li class="review swiper-slide">
+              <blockquote>Eleifend morbi orci velit porttitor sed imperdiet ac ullamcorper Fusce eget tortor vel magna iaculis convallis. Aeneaneros vitae metus pellentesque tincidunt.Cum sociis natoque penatibus et magnis dis parturient montes turpis donec tincidunt tellus.</blockquote>
+              <figure class="app-review-media">
+                <img src="/img/review-image-placeholder.png" alt="Image of PLACEHOLDER" title="PLACEHOLDER">
+                <figcaption>
+                  <h2>Pete "Maverick" Mitchell</h2>
+                  <p>LT, United States Naval Aviator</p>
+                </figcaption>
+              </figure>
+            </li>
+            <li class="review swiper-slide">
+              <blockquote>Eleifend morbi orci velit porttitor sed imperdiet ac ullamcorper Fusce eget tortor vel magna iaculis convallis. Aeneaneros vitae metus pellentesque tincidunt.Cum sociis natoque penatibus et magnis dis parturient montes turpis donec tincidunt tellus.</blockquote>
+              <figure class="app-review-media">
+                <img src="/img/review-image-placeholder.png" alt="Image of PLACEHOLDER" title="PLACEHOLDER">
+                <figcaption>
+                  <h2>Nick "Goose" Bradshaw</h2>
+                  <p>LTJG, Radar Intercept Officer</p>
+                </figcaption>
+              </figure>
+            </li>
+          </ul>
+          <div class="swiper-pagination swiper-pagination-1"></div>
+        </div>
+      </section>
+      <section data-section="appbox-screenshots" id="screens">
+        <h1 class="header-underline">AppBox Screenshots</h1>
+        <ul class="l-flex-container">
+          <li>
+            <figure>
+              <img src="/img/iphone-screenshot.svg" alt="Screenshot 1" title="Screenshot 1">
+            </figure>
+          </li>
+          <li class="screenshot">
+            <figure>
+              <img src="/img/iphone-screenshot.svg" alt="Screenshot 2" title="Screenshot 2">
+            </figure>
+          </li>
+          <li class="screenshot">
+            <figure>
+              <img src="/img/iphone-screenshot.svg" alt="Screenshot 3" title="Screenshot 3">
+            </figure>
+          </li>
+        </ul>
+        <a href="#"><img src="/img/left-arrow-screenshot.svg"></a>
+        <a href="#"><img src="/img/right-arrow-screenshot.svg"></a>
```

Comment:
I would have preferred to see the arrows as backgrounds on the anchors (or pseudo-content inside the anchors.) I don't like seeing graphical decorations in the markup.

There should also be some text content here, either as an `alt` attribute of the anchor or as something like a `<span>` that is hidden for visual mediums. As written, this is not accessible.